docs/design/01-address-scope.md

```diff
@@ -63,7 +63,7 @@ Core originated subnets
   * eOS
   * NX-OS 
 
-Example based on Leafs in QA-DE-1a, assuming qa-de-1 has AZ's a,b,d
+Example based on Leafs in QA-DE-1a, assuming QA-DE-1 has AZ's a,b,d
```

Comment:
qa-de-1 should be consequently written lowercase, at least that's how I like it. Let's make it consistent in this direction.

Reply:
Will do